docs/HowToGuide/production-deployment-guide.md

```diff
@@ -0,0 +1,216 @@
+# API Operator Production Deployment Guide
+
+This documentation will guide you on how to use the Operator on a production scenario.
+Here, we will see how we can deploy a simple API which calls the petstore service 
+([Scenario-2](../../scenarios/scenario-2)) on two different environments (Development and QA).
+
+### Install API Operator
+
+First, install the API Operator. The following command will install the Operator and the configurations
+in the `wso2-system` namespace.
+
+```sh
+>>> apictl install api-operator
+```
+
+Please refer [Install API Operator in CI/CD](install-api-operator-in-cicd.md) for more information
+on installing the API operator.
+
```

Comment:
Without pointing to the doc, let's mention the installation step and point the doc.

Reply:
Updated with fe291a7